dd-sdk-android-gradle-plugin/src/main/kotlin/com/datadog/gradle/plugin/internal/sanitizer/DdUrlSanitizer.kt

```diff
@@ -0,0 +1,41 @@
+package com.datadog.gradle.plugin.internal.sanitizer
+
+import java.util.Locale
+
+/**
+ * It will strip out the user information from the provided URL by applying the following rules:
+ * 1. In case the URL is a valid schema URL : ((ssh|http[s]?):\\) it will strip out the user
+ * information if any from it
+ * 2. In case the URL is a valid schema URL but by mistake the user forgot to add the schema:
+ * [username]:[password]@github.com/(.+).git it will strip out the user information if any and will
+ * return the new url
+ * 3. In case the URL is of a GIT SSH short format: [user@]github.com:(.+).git it will do
```

Comment:
I agree but in the same time after a discussion with @xgouchet we came to the conclusion that leaking the `username` will not be that bad in some situations, otherwise we will have to build a complex RegEx mechanism to threat this case.